TaskPhone/Phone.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+
+namespace TaskPhone
+{
+    public class Phone
+    {
+        private int number;
+        private string model;
+        private int weight;
+
+        public int Number
+        {
```

Comment:
please don't forget to add empty lines after to separate visually your code, so it can be more readable